docs/sources/panels/_index.md

```diff
@@ -4,4 +4,25 @@ type = "docs"
 [menu.docs]
 identifier = "panels"
 weight = 4
-+++
\ No newline at end of file
++++
+
+# Panel overview
+
+The *panel* is the basic visualization building block in Grafana. Each panel has a query editor specific to the data source selected in the panel. The query editor allows you to extract the perfect visualization to display on the panel.
+
+There are a wide variety of styling and formatting options for each panel. Panels can be dragged and dropped and rearranged on the dashboard. They can also be resized.
+
+## Move panels
+
+You can drag and drop panels by clicking and holding the panel title, then dragging it to its new location. You can also easily resize panels by clicking the (-) and (+) icons.
+
+![](/img/docs/animated_gifs/drag_drop.gif)
+
+## Tips and shortcuts
+
+* Click the graph title and in the dropdown menu quickly change span or duplicate the panel.
+* Click the Save icon in the menu to save the dashboard with a new name.
+* Click the Save icon in the menu and then advanced to export the dashboard to json file, or set it as your default dashboard.
+* Click the colored icon in the legend to select series color.
+* Click series name in the legend to hide series.
+* Ctrl/Shift/Meta + click legend name to hide other series.
```

Comment:
I would add 
* Hover over panel and hit `e` key to open panel edit 
* Hover over panel and hit `v` key to open panel in fullscreen view